interfaces/cython/cantera/thermo.pyx

```diff
@@ -809,15 +809,19 @@ cdef class ThermoPhase(_SolutionBase):
         self.state = original_state
         return arr
 
-    def set_equivalence_ratio(self, phi, fuel, oxidizer, basis='mole'):
+    def set_equivalence_ratio(self, phi, fuel, oxidizer, basis='mole', fraction=None, diluent=None):
         """
         Set the composition to a mixture of ``fuel`` and ``oxidizer`` at the
         specified equivalence ratio ``phi``, holding temperature and pressure
         constant. Considers the oxidation of C to CO2, H to H2O and S to SO2.
         Other elements are assumed not to participate in oxidation (that is,
         N ends up as N2). The ``basis`` determines the fuel and oxidizer
         compositions: ``basis='mole'`` means mole fractions (default),
-        ``basis='mass'`` means mass fractions::
+        ``basis='mass'`` means mass fractions. After a mixture given a
+        ``phi`` from ``fuel`` and ``oxidizer`` is  created, it can further
+        be diluted by a ``diluent`` based on a mixing ``fraction``. See also
+        interfaces/cython/cantera/examples/thermo/equivalenceRatio.py for more
```

Comment:
I am not certain about the most appropriate link here, but believe that this should point directly to the [online example](https://cantera.org/examples/python/thermo/equivalenceRatio.py.html) using Sphinx notation (@bryanwweber may have some thoughts). I think it's ok to link to the permanent location as this hopefully will make it into the 2.6 release.